ci/test/testinstall/install_integration_test.go

```diff
@@ -0,0 +1,128 @@
+// +build integration
+
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package testinstall
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/magefile/mage/sh"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	installURL  = "https://raw.githubusercontent.com/mysteriumnetwork/node/master/install.sh"
+	installFile = "install.sh"
+)
+
+// TestInstall tests installation on various Linux distributions.
+// SystemD docker images based on https://github.com/j8r/dockerfiles
+// Installs myst using install.sh, tests for healthcheck output.
+func TestInstall(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
```

Comment:
why not []string?

Reply:
Auto-generated. But right, it could be simplified.